optaplanner-examples/src/main/java/org/optaplanner/examples/meetingscheduling/domain/TimeGrain.java

```diff
@@ -26,7 +26,7 @@
     /**
      * Time granularity is 15 minutes (which is often recommended when dealing with humans for practical purposes).
      */
-    public static final int GRAIN_LENGTH_IN_MINUTES = 15;
+    public static final int GRAIN_LENGTH_IN_MINUTES = 5; // TODO: change it back to 15 / get it from user
```

Comment:
Ah, this kills the score calculation speed :)

I would hope we can leave it set to 15 in optaplanner-examples, but on the other hand, I 've now seen at least 2 real-world cases that need it at 5 minutes. Hmm. Soft recommendation to keep at 15 minutes for optaplanner-examples for now.